.github/DISCUSSION_TEMPLATE/a-setup.yml

```diff
@@ -0,0 +1,28 @@
+title: "[Setup] "
+labels: ["waiting for support"]
+body:
+- type: dropdown
+  id: problem
+  attributes:
+    label: The problem you are facing
+    options:
+      - I need help setting up/connecting the bot
+      - The bot is not connected to Discord
+      - The plugin is not loading/enabling properly
+      - DiscordSRV is showing red in /plugins
+  validations:
+    required: true
+- type: markdown
+  attributes:
+    value: |
+      Please check your log (`logs/latest.log`) for errors from DiscordSRV, during startup. If you cannot find your errors please post your log here. If you were able to find errors regarding the plugin enabling, please post them here.
+
+      If you haven't yet gotten to the step of installing the plugin, please explain what step you're stuck on
+- type: input
+  id: logs
+  attributes:
+    label: Link to your latest.log file
+    description: "You can use [mclo.gs](https://mclo.gs) to hide sensitive information in your log."
+    placeholder: "https://mclo.gs/#####"
+  validations:
+    required: true
```

Comment:
The input here doesn't leave a option to input a error message or a step the user is stuck on